frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -220,7 +232,7 @@ private void inferParentForUnmanagedBranchNotificationAndNotify(String branchNam
 
     Path macheteFilePath = gitRepository.getMacheteFilePath();
     val macheteVFile = VirtualFileManager.getInstance().findFileByNioPath(macheteFilePath);
-    boolean isMacheteFilePresent = macheteVFile != null && macheteVFile.isValid();
+    boolean isMacheteFilePresent = macheteVFile != null && !macheteVFile.isDirectory();
```

Comment:
Maybe `isFile` instead of `!isDirectory`? if such method for checking for _regular file_ is available

Reply:
Unfortunately any method like `isFile` is not available.